config/runtimes/kserve-mlserver.yaml

```diff
@@ -19,10 +19,6 @@ spec:
       env:
         - name: "MLSERVER_MODEL_IMPLEMENTATION"
           value: "{{.Labels.modelClass}}"
-        - name: "MLSERVER_MODEL_NAME"
-          value: "{{.Name}}"
-        - name: "MLSERVER_MODEL_URI"
```

Comment:
`MLServerModelsDirEnv` should be there by default, see [here](https://github.com/kserve/kserve/blob/master/pkg/apis/serving/v1beta1/predictor_sklearn.go#L127)